tests/kit/model_zoo/transformers/opt.py

```diff
@@ -15,10 +15,19 @@ def data_gen():
     attention_mask = torch.zeros((BATCH_SIZE, SEQ_LENGTH), dtype=torch.int64)
     return dict(input_ids=input_ids, attention_mask=attention_mask)
 
+def data_gen_for_causal_lm():
+    # LM data gen
+    # the `labels` of LM is the token of the output, cause no padding, use `input_ids` as `labels`
+    data = data_gen()
+    labels = data['input_ids'].clone()
+    data['labels'] = labels
+    return data
```

Comment:
The old `data_gen` will produce data where all elements are 0. Please refer to the llama file to generate real data.